bom/pom.xml

```diff
@@ -127,6 +127,16 @@
                 <artifactId>jdbi3-noop-cache</artifactId>
                 <version>${dep.jdbi3.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.jdbi</groupId>
+                <artifactId>jdbi3-opentelemetry</artifactId>
+                <version>${dep.jdbi3.version}</version>
+            </dependency>
+            <dependency>
```

Comment:
can we make that a separate PR? (Yeah, I know...)

Reply:
Yeah, I should have done two PRs. I'll do that next time.